test/controllers/WhatDoYouWantToDoControllerSpec.scala

```diff
@@ -227,7 +246,7 @@ class WhatDoYouWantToDoControllerSpec extends BaseSpec with JsoupMatchers with B
         status(result) mustBe OK
 
         doc.title() must include(Messages("your.paye.income.tax.overview"))
-        doc.body().toString must include(Messages("tai.WhatDoYouWantToDo.ChangedTaxCode"))
+        doc.body().toString must include(Messages("tai.WhatDoYouWantToDo.ChangedTaxCode", mostRecentTaxCodeChangeDate))
```

Comment:
is there a check for this to no include taxCodeChange banner? If so is it worth adding one in?